From cd8d0fe14c04e30111e94c7f6f5b0c69779ad942 Mon Sep 17 00:00:00 2001
From: "Eduardo Lima (Etrunko)" <etrunko@redhat.com>
Date: Thu, 21 Sep 2017 10:16:56 +0200
Subject: [PATCH] doc: Adjust reference to spice-gtk man page for remote-viewer

Similar to last commit, as noticed by reporter in bugzilla:

https://bugzilla.redhat.com/show_bug.cgi?id=1477966

Man page should reference spice-client, not spice-gtk.

Signed-off-by: Eduardo Lima (Etrunko) <etrunko@redhat.com>
Acked-by: Victor Toso <victortoso@redhat.com>
---
 man/remote-viewer.pod | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/remote-viewer.pod b/man/remote-viewer.pod
index 60ea3a6..560f73f 100644
--- a/man/remote-viewer.pod
+++ b/man/remote-viewer.pod
@@ -385,6 +385,6 @@ to the extent permitted by law.
 
 =head1 SEE ALSO
 
-C<virt-viewer(1)>, C<spice-gtk(1)>, the project website C<http://virt-manager.org>
+C<virt-viewer(1)>, C<spice-client(1)>, the project website C<http://virt-manager.org>
 
 =cut
-- 
2.14.3

